### PR TITLE
Removes Flechette Ammo

### PR DESCRIPTION
**Pull request by Hathkar**
## About The Pull Request

Removes flechette ammo from the game, and all references to it from weapons and crate descriptions.

## Why It's Good For The Game

Flechette ammo is agonizingly hard to balance with our armor system, as it's either useless or overpowered.

## Changelog
:cl:
del: Removed flechette ammo
tweak: Tweaked some item descriptions that referred to flechette.
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8122:error: undefined type: /obj/item/ammo_magazine/shotgun/flechette

_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8123:error: undefined type: /obj/item/ammo_magazine/shotgun/flechette

_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8131:error: unknown type

_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8131:error: unknown type

**Comment by spookydonut** [comment]
_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8122:error: undefined type: /obj/item/ammo_magazine/shotgun/slug

_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8123:error: undefined type: /obj/item/ammo_magazine/shotgun/slug

_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8131:error: unknown type

_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm:8131:error: unknown type

**Comment by psykzz** [comment]
This is conflicting with the massive balance PR since everything got reworked.
Hold off TM / merge until that has been resolved.

File: code/__DEFINES/mode.dm [hunk 1]
```diff
@@ -83,7 +83,6 @@
 
 #define SURVIVOR_WEAPONS list(\
 				list(/obj/item/weapon/gun/smg/mp7, /obj/item/ammo_magazine/smg/mp7),\
-				list(/obj/item/weapon/gun/shotgun/double/sawn, /obj/item/ammo_magazine/shotgun/flechette),\
```
Comment (spookydonut) [inline code/__DEFINES/mode.dm:86]: why are you removing the sawn off from survivors

File: tgmc.dme [hunk 2]
```diff
@@ -1,1465 +1,1465 @@
...
-#include "code\modules\vehicles\multitile\multitile.dm"
-#include "code\modules\vehicles\multitile\tank.dm"
-#include "code\modules\vehicles\multitile\tankvendor.dm"
-#include "code\modules\xenomorph\xeno_structures.dm"
-#include "interface\interface.dm"
-#include "interface\menu.dm"
-#include "interface\stylesheet.dm"
-#include "interface\skin.dmf"
-// END_INCLUDE
+// DM Environment file for baystation12.dme.
```
Comment (spookydonut) [inline tgmc.dme:1]: line endings

Comment (spookydonut) [reply tgmc.dme:1]: still line endings, don't self resolve please.

**Comment by psykzz** [comment]
@Hathkar since most of the big changes have gone through, do you still think this is needed? 
If not can you close. 

If you do want to continue with this, you'll need to fix the conflicts and get it approved by @MerryweatherLost or generally via balance.

**Comment by psykzz** [comment]
No reply, so im closing for now, feel free to poke me about reopening it @Hathkar

**Pull request closed without merging**